build.maven/iar/pom.xml

```diff
@@ -14,7 +14,7 @@
 
 	<properties>
 		<antrun.version>1.7</antrun.version>
-		<project.build.plugin.version>8.0.3</project.build.plugin.version>
+		<project.build.plugin.version>8.0.4-SNAPSHOT</project.build.plugin.version>

```

Comment:
Can you now use SNAPSHOT version here? 
Do you update later to 9.1.0-SNAPSHOT?

Reply:
parent poms are resolvable (so snapshots repos are public visible)
https://repo.axonivy.com/artifactory/libs/ch/ivyteam/ivy/project/build.maven.iar/

and as far as I know we no longer manipulate the final pom 's so it should work.

Reply:
and yes the final solution is to use 9.1.0-snap